### PR TITLE
Add V2020-02-02-Preview ShortTermRetention API Specification of SQL DB

### PR DESCRIPTION
**Pull request by lululilliancoding**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/lululilliancoding/azure-rest-api-specs/dev%2Flillian%2Fdev-sql-Microsoft.Sql-2020-02-02-preview...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
    
fix https://github.com/Azure/azure-rest-api-specs-pr/issues/9769

### CODE REVIEW

**Comment by lululilliancoding** [comment]
Hi I don't understand this error. 
image
This is new version API (2020-02-02) but based on old version API (2017-10-01). Except ShortTermRetentioPolicy.cs was updated, all other files are same as 2017 API version. Why are they "WRITEONLY_PROPERTY_NOT_ALLOWED_IN_RESPONSE"?

**Comment by lululilliancoding** [comment]
OK I removed all non-related json files and some checks are passed. Investigating other errors.

**Comment by lululilliancoding** [comment]
image
All checks passed. Please take a review and let me know if any other issue. Thanks.

**Comment by lululilliancoding** [comment]
Anyone from ARM team can take a look at this PR? Jared is moving out of SQL DB and no longer reviewing this PR.

File: specification/sql/resource-manager/readme.md [hunk 1]
```diff
@@ -24,13 +24,23 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2020-02
```
Comment (jamestao) [inline specification/sql/resource-manager/readme.md:27]: This should always point to composite package. if you want your APIs included, add them to the composite package.

**Review by jamestao — CHANGES_REQUESTED** [review]

File: specification/sql/resource-manager/readme.md (hunk 1 above)
Comment (lululilliancoding) [reply specification/sql/resource-manager/readme.md]: Modified. Please check.

**Review by jamestao — APPROVED** [review]

**Comment by lululilliancoding** [comment]
@xccc-msft Hi Chuang, please start review.

**Review by xseeseesee — APPROVED** [review]

**Comment by lululilliancoding** [comment]
@xccc-msft Do you know why the PR is still blocked of merging? What further action can I provide?

**Comment by xccc-msft** [comment]
@lululilliancoding ARM review is required. I'm not sure why no update from ARM end...

**Comment by lululilliancoding** [comment]
@xccc-msft You are not from ARM reviewers group?

**Review by majastrz — APPROVED** [review]
Delta from `2017-10-01-preview` on this API looks good to me. Signed off from ARM side.

**Comment by lululilliancoding** [comment]
Why the merging is still blocked?

**Comment by majastrz** [comment]
Only the SDK team has permissions to merge a PR. @xccc-msft is the assigned reviewer from that team and can help with merging.

**Comment by lululilliancoding** [comment]
Oh I see. 
So the process is Service team signoff -> ARM team signoff -> SDK team merge. 
Do I have to manually 'at' members from ARM team from oncall list like this time? @majastrz

**Comment by majastrz** [comment]
Yes, for quickest response it's best to at the on-call person.

**Comment by lululilliancoding** [comment]
ACK. Thank you for the info.

**Pull request merged**